mod_reforged/hooks/config/character_names.nut

```diff
@@ -100,6 +100,7 @@
 	"the Blade of %factionname% ",
 	"the Singing Blade"
 ];
+
```

Comment:
This extra line has nothing to do with the changes in this PR and should not be included.